### PR TITLE
Update web-platform-tests

### PR DESCRIPTION
**Pull request by nox**
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/19738)

### CODE REVIEW

**Comment by nox** [comment]
I wonder if something went wrong on Buildbot, that's a lot of timeouts…

@bors-servo retry

**Comment by jdm** [comment]
Last time I saw results like that, we had an http server interfering with the wpt one.

File: tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/browsers/servo.py [hunk 1]
```diff
@@ -54,7 +54,7 @@ def env_extras(**kwargs):
 def env_options():
     return {"host": "127.0.0.1",
             "external_host": "web-platform.test",
-            "bind_hostname": "false",
+            "bind_hostname": "true",
```
Comment (jdm) [inline tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/browsers/servo.py]: This change got reverted. It probably should not have been folded into the previous big wptup commit.

**Comment by nox** [comment]
tfw you forgot to `git add` the new expectations.

@bors-servo try

**Pull request merged**